packages/fx-core/src/common/tools.ts

```diff
@@ -495,3 +495,35 @@ export async function copyFiles(
     });
   }
 }
+
+export function getSiteNameFromResourceId(resourceId: string): string {
+  const result = parseFromResourceId(
```

Comment:
is there any method in azure sdk to parse resourceId? as for the strings do we also consider case insensitive?

Reply:
Seems not. I also updated the regex pattern to use `[^\/]` so we don't need to change code if azure naming rule changes.

Reply:
I think wei's concern is whether the keys are case sensitive, like 'resourceGroups' 'providers\/Microsoft.Web\/sites'.

Reply:
The resource Id is get from arm deployment. We haven't found situation that the letter case changes.

Reply:
Can we use case insensitive match? It's a quick change (adding the `i` flag) which makes our life easier if Azure API behavior changes.